src/common/navigation.yaml

```diff
@@ -109,7 +109,6 @@ sections:
         usePageTitles: true
         pages:
           - path: /blockstack-cli
-          - path: /language-clarity
```

Comment:
ah, that reminds me: we should add a redirect for this old URL - otherwise old links will result in 404. You'd need to add one redirect object in here: https://github.com/blockstack/docs/blob/master/next.config.js#L281